mmv1/templates/terraform/examples/base_configs/test_file.go.erb

```diff
@@ -15,6 +15,12 @@ import (
 <% unless object.skip_delete -%>
   "github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 <% end -%>
+
+<% if version == 'private' -%>
+    transport_tpg "internal/terraform-next/google-private/transport"
```

Comment:
Can we pull this if/else block into a function in ruby somewhere? Something like <%= transport_tpg_import(version) -%> should be usable anywhere we have a .erb file

Reply:
Later on, there will be more package import.
For example, add  "internal/terraform-next/google-private/acctest" in the test_file.go.erb file, and add "internal/terraform-next/google-private/tpgresource" in the resource.erb file. 

Do we need to put each import in its own function, like tpg_acctest_import, tpg_resource_import?

Reply:
Hmmmm, I'm at least thinking we should consolidate the shared part. It could look like:
`package_tpg "<%= import_path(version) -%>/package"

Reply:
That makes sense. Made the change.